packages/dds/tree/src/feature-libraries/forest-proxy/forestProxy.ts

```diff
@@ -0,0 +1,103 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { FieldKey } from "../../tree";
+import { IEditableForest, TreeNavigationResult, ITreeSubscriptionCursor } from "../../forest";
+
+export const proxySymbol = Symbol("forest-proxy");
+
+class TargetForest {
+	private readonly _cursor: ITreeSubscriptionCursor;
```

Comment:
since this is readonly and there is a public getterwith a matching type, why not just make it a public field?

Reply:
That's a leftover from my experiments.